### ISSUE TITLE
Issue loading pyBigWig within conda environment

### ISSUE THREAD

**Issue by kevinrue** [dpryan79/pyBigWig#44]
Hi,

I've seen a few issues resolved that related to libcurl. However, I can't find a solution to my current issue. Does that look any familiar?

```
$ python
Python 3.6.2 | packaged by conda-forge | (default, Jul 23 2017, 22:59:30) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyBigWig
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /gfs/devel/kralbrecht/miniconda3/envs/py362/lib/libcurl.so.4: undefined symbol: libssh2_scp_recv2
```

I've tried installing `pyBigWig` using both `pip` and `conda install`. Both of them report a successful installation, yet result in the same error.
In both cases I seem to have the latest version installed:
```
pybigwig                  0.3.5               np112py36_0    bioconda
```

Let me know if you need additional details about the system (*e.g.*, other package versions installed in the environment). I've updated them all the latest, using `conda update --all`, but that didn't solve the issue.

Many thanks in advance,
Kevin

**Comment by kevinrue** [comment]
_Somehow_, I got it to work (I mean, loading `pyBigWig`).

Following the advice of a seemingly different issue [here](https://github.com/archlinuxfr/yaourt/issues/86) ...

> Anaconda's curl has troubles when working with https, so that might be the reason why yaourt can't install anything. The command "conda remove curl" leaves you with normal version of curl,

... the last few commands that I ran were:

```
$ conda remove curl
Fetching package metadata .............
Solving package specifications: .

Package plan for package removal in environment .../miniconda3/envs/py362:

The following packages will be REMOVED:

    curl:     7.54.1-0          conda-forge
    pybigwig: 0.3.5-np112py36_0 bioconda   

Proceed ([y]/n)? y
```

```
$pip uninstall pyBigWig
$pip install pyBigWig
```

which then loaded `pyBigWig` like a (Py)charm:

```
>>> import pyBigWig
>>> 
```

For the record, my current environment does _not_ include Anaconda's `curl` anymore, here is the full list of packages installed right now (while I can import `pyBigWig` successfully):

```
$ conda list
# packages in environment at .../miniconda3/envs/py362:
#
alabaster                 0.7.10                    <pip>
alignlib-lite             0.3                       <pip>
Babel                     2.5.1                     <pip>
biopython                 1.70                      <pip>
blas                      1.1                    openblas    conda-forge
bleach                    2.0.0                     <pip>
bokeh                     0.12.9                    <pip>
brewer2mpl                1.4.1                     <pip>
bx-python                 0.7.3                     <pip>
ca-certificates           2017.7.27.1                   0    conda-forge
certifi                   2017.7.27.1              py36_0    conda-forge
CGATReport                0.7.6.1                   <pip>
cycler                    0.10.0                    <pip>
Cython                    0.26.1                    <pip>
decorator                 4.1.2                     <pip>
docutils                  0.14                      <pip>
entrypoints               0.2.3                     <pip>
et-xmlfile                1.0.1                     <pip>
future                    0.16.0                    <pip>
ggplot                    0.11.5                    <pip>
html5lib                  0.999999999               <pip>
httplib2                  0.10.3                    <pip>
ipykernel                 4.6.1                     <pip>
ipython                   6.2.0                     <pip>
ipython-genutils          0.2.0                     <pip>
ipywidgets                7.0.1                     <pip>
jedi                      0.11.0                    <pip>
Jinja2                    2.9.6                     <pip>
jsonschema                2.6.0                     <pip>
jupyter                   1.0.0                     <pip>
jupyter-client            5.1.0                     <pip>
jupyter-console           5.2.0                     <pip>
jupyter-core              4.3.0                     <pip>
krb5                      1.14.2                        0    conda-forge
libgfortran               3.0.0                         1  
libssh2                   1.8.0                         1    conda-forge
MarkupSafe                1.0                       <pip>
matplotlib                2.0.2                     <pip>
matplotlib-venn           0.11.5                    <pip>
mistune                   0.7.4                     <pip>
mkl                       2017.0.3                      0  
mpld3                     0.3                       <pip>
nbconvert                 5.3.1                     <pip>
nbformat                  4.4.0                     <pip>
ncurses                   5.9                          10    conda-forge
networkx                  1.11                      <pip>
nose                      1.3.7                     <pip>
notebook                  5.1.0                     <pip>
numpy                     1.12.1          py36_blas_openblas_200  [blas_openblas]  conda-forge
numpy                     1.13.1                    <pip>
olefile                   0.44                      <pip>
openblas                  0.2.19                        2    conda-forge
openpyxl                  2.5.0a3                   <pip>
openssl                   1.0.2l                        0    conda-forge
pandas                    0.20.3                    <pip>
pandocfilters             1.4.2                     <pip>
parso                     0.1.0                     <pip>
pep8                      1.7.0                     <pip>
pexpect                   4.2.1                     <pip>
pickleshare               0.7.4                     <pip>
Pillow                    4.2.1                     <pip>
pip                       9.0.1                    py36_0    conda-forge
prompt-toolkit            1.0.15                    <pip>
psycopg2                  2.7.3.1                   <pip>
ptyprocess                0.5.2                     <pip>
pybedtools                0.7.10                    <pip>
pyBigWig                  0.3.5                     <pip>
Pygments                  2.2.0                     <pip>
pyparsing                 2.2.0                     <pip>
pysam                     0.11.1                    <pip>
python                    3.6.2                         0    conda-forge
python-dateutil           2.6.1                     <pip>
pytz                      2017.2                    <pip>
pyzmq                     16.0.2                    <pip>
qtconsole                 4.3.1                     <pip>
rdflib                    4.2.2                     <pip>
readline                  6.2                           0    conda-forge
requests                  2.18.4                    <pip>
rpy2                      2.9.0                     <pip>
ruffus                    2.6.3                     <pip>
scikit-learn              0.19.0                    <pip>
scipy                     0.19.1                    <pip>
seaborn                   0.8.1                     <pip>
setuptools                36.3.0                   py36_0    conda-forge
setuptools                36.5.0                    <pip>
simplegeneric             0.8.1                     <pip>
six                       1.11.0                    <pip>
Sphinx                    1.6.3                     <pip>
sphinxcontrib-programoutput 0.11                      <pip>
sphinxcontrib-websupport  1.0.1                     <pip>
SQLAlchemy                1.1.14                    <pip>
sqlite                    3.13.0                        1    conda-forge
statsmodels               0.8.0                     <pip>
terminado                 0.6                       <pip>
testpath                  0.3.1                     <pip>
tk                        8.5.19                        2    conda-forge
tornado                   4.5.2                     <pip>
traitlets                 4.3.2                     <pip>
urllib3                   1.22                      <pip>
wcwidth                   0.1.7                     <pip>
webencodings              0.5.1                     <pip>
weblogo                   3.5.0                     <pip>
wheel                     0.30.0                     py_1    conda-forge
widgetsnbextension        3.0.3                     <pip>
xlwt                      1.3.0                     <pip>
xz                        5.2.3                         0    conda-forge
zlib                      1.2.11                        0    conda-forge
```

Also, for the sake of completeness, I can confirm that all traces of `libcurl` have disappeared from 
`.../miniconda3/envs/py362/lib` after `conda remove curl`. Before the `remove`, those files were present in the folder:

```
libcurl.a
libcurl.la
libcurl.so
libcurl.so.4
libcurl.so.4.4.0
```

Bottom line is that the other issue seems to have nailed the source of the issue: Ananconda's curl. Nothing to fix in pyBigWig as far as I can see.

**Comment by dpryan79** [comment]
Weird, there are days when I regret adding support for remote files. Anyway, I'm glad you got things working.
